src/applications/vaos/actions/appointments.js

```diff
@@ -129,21 +143,20 @@ async function getClinicDataBySystem(facilityClinicListMap) {
 async function getAdditionalFacilityInfo(futureAppointments) {
   // Get facility ids from non-VA appts or requests
   const requestsOrNonVAFacilityAppointments = futureAppointments.filter(
-    appt => !appt.clinicId,
+    appt => !getVARClinicId(appt),
   );
   let facilityIds = requestsOrNonVAFacilityAppointments
     .map(appt => appt.facilityId || appt.facility?.facilityCode)
     .filter(id => !!id);
 
   // Get facility ids from VA appointments
   const vaFacilityAppointments = futureAppointments.filter(
-    appt => appt.clinicId,
+    appt => !!getVARClinicId(appt),
```

Comment:
Similar to above, you could use videoType and isCommunityCare to make this check more clear.